graphql_compiler/query_formatting/common.py

```diff
@@ -44,37 +53,35 @@ def _deserialize_anonymous_json_argument(expected_type, value):
             GraphQLDateTime: datetime.datetime with tzinfo=pytz.utc
             GraphQLFloat: float
             GraphQLDecimal: decimal.Decimal
-            GraphQLInt: six.integer_types, supporting long integers
-            GraphQLString: six.string_types
+            GraphQLInt: int
+            GraphQLString: str
             GraphQLBoolean: bool
-            GraphQLID: six.string_types
+            GraphQLID: str
 
     Raises:
         ValueError if the value is not appropriate for the type. ValueError is chosen because
         it is already the base case of exceptions raised by the GraphQL parsers.
     """
     allowed_types_for_graphql_type = {
-        GraphQLDate.name: (six.string_types,),
-        GraphQLDateTime.name: (six.string_types,),
-        GraphQLFloat.name: (six.string_types, float, six.integer_types),
-        GraphQLDecimal.name: (six.string_types, float, six.integer_types),
-        GraphQLInt.name: (six.integer_types, six.string_types),
-        GraphQLString.name: (six.string_types,),
+        GraphQLDate.name: (str,),
```

Comment:
Mypy was going haywire with the nested type tuples of varying depth. So I just got rid of "six.string_types" and "six.string_types" since we longer support python2 and replaced them with the python3 only equivalents.